recipes/zulu-openjdk/all/conanfile.py

```diff
@@ -22,14 +23,19 @@ def _jni_folder(self):
         return os.path.join("include", folder)
 
     def configure(self):
```

Comment:
This method is doing validation checks, and should thus be named `validate`.
I would make the logic similar to that of `wasmtime`.
Do something like:
```python
        try:
            self.conan_data["sources"][self.version][str(self.settings.os)][str(self.settings.arch)]
        except KeyError:
            raise ConanInvalidConfiguration("Binaries for this combination of architecture/version/os are not available")
```

Reply:
And then call validate from configure? But this makes no different. ..
also, the suggested change would be misleading, there are binaries for Linux arm, for example, but no one has added them so far....